quickstarts/host-inventory-quickstart.yaml

```
@@ -21,9 +21,29 @@ spec:
 
 
   tasks:
-    - description: >-
+    - description: |-
 
-        1.  Configure your host inventory settings. View [documentation](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html-single/clusters/index#enable-cim) to follow the steps to configure your host inventory settings.
+        1.  Configure your host inventory settings.
+
+          a. Go to **Host inventory**.
+
+          b. Select **Configure host inventory settings**.
+
+        1. In the 'Configure host inventory settings' modal that opens, 
+
+          a. Enter the required values for Database storage, System Storage, and Image Storage.
+
+          **Note**: The modal is for connected environments only. For disconnected environments, follow the steps in the [configuring your host inventory settings documentation](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html-single/clusters/index#enable-cim).
+
+          b. If your cluster is running on AWS and you have not configured your load balancer, select the 'Configure load balancer on Amazon Web Services for me' checkbox.
+
+        1. Then click configure.
+        
+          **Note**: Storage sizes cannot be changed after they are configured. Also, configuring the load balancer on AWS cannot be reverted.
```

Should this be two separate notes? If not I would remove the "Also"
```suggestion
          **Note**: Storage sizes cannot be changed after they are configured. 
           **Note**: Configuring the load balancer on AWS cannot be reverted.
```